### PR TITLE
Chore/page no index

### PR DESCRIPTION
**Pull request by rsdeus**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -18,8 +18,8 @@ def prepare_database(app_path)
   # Remove previous existing db, and recreate one.
   disable_docker_compose = ENV.fetch("DISABLED_DOCKER_COMPOSE", "false") == "true"
   unless disable_docker_compose
-    system("docker-compose -f docker-compose.yml down -v")
-    system("docker-compose -f docker-compose.yml up -d --remove-orphans")
+    system("sudo docker-compose -f docker-compose.yml down -v")
+    system("sudo docker-compose -f docker-compose.yml up -d --remove-orphans")
```
Comment (froger) [inline Rakefile:22]: docker should never have to be run as `sudo` :)

File: app/overrides/profiles_noindex.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+##
+views = ["decidim/profiles/show", 
+         "decidim/user_activities/index",
+         "decidim/user_timeline/index",
+         "decidim/user_conversation/index"].freeze
+
+views.each do |view|
+  Deface::Override.new(:virtual_path => view,
+                       :name => "profiles_noindex",
+                       :insert_after => "erb[loud]:contains('cell')",
```
Comment (froger) [inline app/overrides/profiles_noindex.rb]: Not sure it will insert only once, case you have two `cell` used.

File: Rakefile (hunk 1 above)
Comment (rsdeus) [reply Rakefile:22]: oh sorry and thanks, I forgot. It's an trick for an issue from my installation. :)

File: app/overrides/profiles_noindex.rb (hunk 2 above)
Comment (rsdeus) [reply app/overrides/profiles_noindex.rb]: OK, I've improved the criteria for `:insert_after => "erb[loud]:contains('cell \"decidim/profile\"')",` ;)

**Pull request merged**